plugins/hypervisors/kvm/src/main/java/com/cloud/hypervisor/kvm/resource/LibvirtVMDef.java

```diff
@@ -1613,6 +1613,10 @@ public String toString() {
     }
 
     public static class VideoDef {
+
+        public static final String VIDEO_MODEL = "video.hardware";
+        public static final String VIDEO_RAM = "video.ram";
```

Comment:
@leolleeooleo you can re-use the constants at VmDetailConstants (to use in LibvirtComputingResource), no need of this constants.